composer.json

```diff
@@ -16,17 +16,17 @@
 
     "require": {
         "php": "^7.1",
-        "symfony/console": "^4.1",
-        "symfony/dependency-injection": "^3.0 || ^4.1.12",
-        "symfony/process": "^3.0 || ^4.0",
+        "symfony/console": "^4.1 || ^5.0.4",
```

Comment:
Is there a reason to use `^5.0.4` instead of just `^5.0`?

Reply:
It's probably not a good idea. I specified this version because it's the last PATCH version.

Reply:
So you're going to update this every time Symfony releases another patch version?

Reply:
Of course not. So, I removed the PATCH version from the constraint.